core/embed/rust/src/ui/model_tt/bootloader/intro.rs

```diff
@@ -44,11 +47,11 @@ impl<'a> Intro<'a> {
             bg: Pad::with_background(BLD_BG),
```

Comment:
Most of the time we are using little different `with_...` method appearance:

```rs
pub fn with_clear(mut self) -> Self {
    self.clear = true;
    self
}
```
instead of 
```rs
pub fn with_clear(self) -> Self {
    Self {
        clear: true,
        ..self
    }
}
```

Reply:
yeah but this one is better :) reason being, the latter can be `const` while the former can't

Reply:
Does it mean that it would be beneficial for the codesize to use it in all the places and make them all `const`?

Reply:
You can try. But the last time I tried this the saving was marginal, on the order of single bytes.

This makes sense when you need a const value for something (and so your `new()` is also const), otherwise it's just a matter of style IMO.